config/system-stats-monitor.json

```diff
@@ -15,5 +15,12 @@
 		"includeAllAttachedBlk": true,
 		"lsblkTimeout": "5s"
 	},
+	"host": {
+		"metricsConfigs": {
+			"host/uptime": {
+				"displayName": "host/uptime"
```

Comment:
This is because I think in future, we *may* allow user to specify more options for each metrics.

For example (I'm really guessing here :P so the examples may not be appropriate):

- whether the metric should have additional tags
- whether the metric should be gauge or counter
- whether the metric should be reported to Prometheus or some other monitoring backend.

Having the `displayName` key, allows people to add more fields. If we use a simplay key-value pair map, I'm afraid that we might have trouble to keep backward compatibility in future.

WDYT?